pages/servers.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title:  "Servers"
+permalink: /servers/
+---
+
+List of servers (in alphabetical order) that provide a CCTray feed:
+
+### Buddybuild
+
+* Server home: <https://www.buddybuild.com/>
+* CCMenu setup instructions: <http://docs.buddybuild.com/docs/ccmenu>
```

Comment:
I know CCTray wouldn't exist without CCMenu but should this page be tool agnostic and we create a separate page for tools that can consume CCTray?

Reply:
Don't know. I was more thinking about the server people. Now they could put the information about their server in one place. If we ask them to put setup instructions on tool-specific pages, I worry they won't do it at all.

Reply:
I wouldn't expect the server people to be adding the tool-specific set-up. They should just be providing the generic instructions, this is the feed URL, this is how you can auth etc.

The tool makers should be adding their own entries and any specific steps for setting up servers if it wasn't obvious.

I just like the idea of having a list of consumers / tools so it's easy for developers to see the options they have. It would also be a good way to promote the format "hey, look at all these tools that already exist!" etc

I'm not suggesting it needs to be done as part of this pull request. I'm happy to open a new issue to create and source the initial list of tools.

Reply:
In the past the CI server makers did provide specific instructions for CCMenu. Maybe, we can leave it as is for now, and address later?

Reply:
I'm happy to leave this for now and think about it some more later 👍